service/src/main/java/com/epam/esm/services/service/EntityService.java

```diff
@@ -20,4 +21,23 @@
     Collection<T> findAll() throws ServiceException;
 
     Collection<T> findBy(EntityFinder<T> entityFinder) throws ServiceException;
+
+    default AscDesc parseAscDesc(String param) throws BadRequestException {
```

Comment:
Bad place for sorting logic in interface of Entity.

Unify search logic to one approach:

boolean - true/false or  string values - acs/desc.

Reply:
do you mean add AscDesc parseAscDesc(boolean param) {} ?
Don't we recieve all parameters as String? I considered it will be more universal and convenient to handle either asc\desc or true\false or 1\2. May I just delete 1 and 2 cases?
I don't know, I can move this logic back to AscDesc enum.